neuralop/models/padding.py

```diff
@@ -54,23 +54,37 @@ def pad(self, x):
             
             print(f'Padding inputs of {resolution=} with {padding=}, {self.padding_mode}')
 
+            out_put_pad = padding
```

Comment:
```suggestion
            output_pad = padding
```
Let's just call it `output_pad`